### PR TITLE
Nit from PR #8344 missed due to autosubmit. 

### PR DESCRIPTION
**Pull request by KoolJBlack**
https://github.com/google/iree/pull/8344

### CODE REVIEW

File: iree/compiler/Codegen/LLVMGPU/Verifiers.cpp [hunk 1]
```diff
@@ -191,9 +191,6 @@ LogicalResult verifyGPUMatmulTensorCorePipeline(
     bytesSize = type.getElementType().getIntOrFloatBitWidth() / 8;
   } else if (RankedTensorType type = inputType.dyn_cast<RankedTensorType>()) {
     bytesSize = type.getElementType().getIntOrFloatBitWidth() / 8;
-  } else if (UnrankedTensorType type =
```
Comment (MaheshRavishankar) [inline iree/compiler/Codegen/LLVMGPU/Verifiers.cpp:194]: drive by comment unrelated to PR. Will we ever see an `UnrankedTensorType` here

Comment (ThomasRaoux) [inline iree/compiler/Codegen/LLVMGPU/Verifiers.cpp:194]: actually you can do something even simpler you can do:
`inputType.cast<ShapedType>().getElementType();`
Matmul operand has to be shaped type so it is fine to do the cast directly.

Comment (ThomasRaoux) [reply iree/compiler/Codegen/LLVMGPU/Verifiers.cpp:194]: This way you don't need separate case for memref and tensor.

Comment (ThomasRaoux) [reply iree/compiler/Codegen/LLVMGPU/Verifiers.cpp:194]: also please update the the simt verifier as well.

**Review by ThomasRaoux — APPROVED** [review]

**Comment by benvanik** [comment]
(please squash when merging in the future so we don't end up with 'thomas nit' commits in the main history :)
image

**Pull request merged**